std/conv.d

```diff
@@ -3967,7 +3967,7 @@ private void emplaceInitializer(T)(ref T chunk) @trusted pure nothrow
     else
     {
         import core.stdc.string : memcpy;
-        static immutable T init = T.init;
+        static immutable T init;// = T.init;
```

Comment:
* `TypeInfo.init` is deprecated and should be replaced by `TypeInfo.initializer`  in all code that try to use it.
* `TypeInfo.`... is not a good idea for such a core functuonality such as emplace, because it is not usable from `version (FreeStanding)` code.
* `T init = T.init` is better than `static immutable`, because:
  + `static immutable` keeps objects alive for ever.
  + stack allocated data is more cache friendly